internal/models/users/user_quiz/user_quiz.go

```diff
@@ -95,24 +96,37 @@ func NextQuestionInQuiz(db *sql.DB, userID uuid.UUID, quizID uuid.UUID) (*QuizDa
 	if err != nil || !partialQuiz.Published || partialQuiz.QuestionNumber == 0 {
 		return nil, ErrNoSuchQuiz
 	}
-	nextQuestion, err := StartNextQuestion(db, userID, quizID)
+	nextQuestionAndSeconds, err := startNextQuestion(db, userID, quizID)
 	if err != nil {
 		return nil, err
 	}
+
+	pointsSoFar, err := getPointsGatheredInQuiz(db, quizID, userID)
+	if err != nil {
+		return nil, err
+	}
+
 	return &QuizData{
 		*partialQuiz,
-		*nextQuestion,
+		*nextQuestionAndSeconds.question,
+		pointsSoFar,
+		nextQuestionAndSeconds.secondsLeft,
 	}, nil
 
 }
 
+type QuestionAndSecondsLeft struct {
```

Comment:
Is there a benefit to creating a struct for this, instead of just returning multiple separate values from `startNextQuestion`?

Reply:
no there isnt one, let me fix